bindings/requests.go

```diff
@@ -17,8 +17,10 @@ type OperationKind string
 
 // Non exhaustive list of operations. A binding can add operations that are not in this list.
 const (
-	GetOperation    OperationKind = "get"
-	CreateOperation OperationKind = "create"
-	DeleteOperation OperationKind = "delete"
-	ListOperation   OperationKind = "list"
+	GetOperation      OperationKind = "get"
+	CreateOperation   OperationKind = "create"
+	DeleteOperation   OperationKind = "delete"
+	ListOperation     OperationKind = "list"
+	QueryOperation    OperationKind = "query"
```

Comment:
Late catch: Component specific operation constants should not go here. Please move them to `grapnel.go`.